src/Makefile.am

```diff
@@ -319,6 +321,7 @@ BITCOIN_CORE_H = \
   util/result.h \
   util/serfloat.h \
   util/signalinterrupt.h \
+  util/strencodings.h \
```

Comment:
In commit "build: Introduce libbitcoin_kernel as an internal compilation unit" (1f98ed5e56230a61b699f669459e39fa101a1ca5)

Reason for adding strencodings.h this commit? Can it be explained in the commit description, or does it maybe belong the previous commit?